src/cc/libclient/KfsClient.cc

```diff
@@ -4987,7 +5025,7 @@ KfsClientImpl::ComputeFilesize(kfsFileId_t kfsfid)
     time_t const endTime   = startTime + mRetryDelaySec * mMaxNumRetriesPerOp;
     for (int retry = 0; ; retry++) {
         lop.lastChunkOnlyFlag = true;
-        GetLayout(lop);
+        GetLayout(lop, 0);
```

Comment:
C NULL
C++ 0
C++11 std::nullptr

The intention is to allow to compile C++, hence 0.

Netscape / mozilla had portability guidelines prior to moving mozilla project to C++11, agued in favor of 0, and that looked convincing enough to me. If I remember right, the main reason due to use of C macro sometimes defined NULL as (void*)0, which I also encountered in the past, and the argument against using or at least minimizing use of C preprocessor in C++ code.
Well here is another fragment never ending topic :)
http://www.cplusplus.com/forum/beginner/5604/

The use of if (ptr) and if (! ptr) is more convincing due to smart pointer operator overload, it follows then that NULL should not be used in C++ code at all.